socorro/processor/rules/mozilla.py

```diff
@@ -56,6 +71,17 @@ def action(self, raw_crash, dumps, processed_crash, processor_meta):
                         f"{raw_key} has non-1 value"
                     )
 
+            elif value_type == "int":
+                try:
+                    processed_crash[processed_key] = int(raw_crash[raw_key])
+                except ValueError:
+                    processor_meta["processor_notes"].append(
+                        f"{raw_key} has a non-int value"
+                    )
+
+            elif value_type == "string":
+                processed_crash[processed_key] = raw_crash[raw_key]
+
```

Comment:
There are probably some rules we can remove and move the annotation moving here. I'll think about that later.